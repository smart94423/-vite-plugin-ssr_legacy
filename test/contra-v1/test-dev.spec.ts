import { beforeAll } from 'vitest'
import { createServer } from 'vite'
import { test } from './test'

beforeAll(async () => {
  await devApp()
}, 10 * 1000)

test()

async function devApp() {
  await createServer({
    root: __dirname,
    server: { middlewareMode: true }
  })
}
