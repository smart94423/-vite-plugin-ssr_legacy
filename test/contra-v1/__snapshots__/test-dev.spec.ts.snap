// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contra Simulation > private properties 1`] = `"<script id=\\"vite-plugin-ssr_pageContext\\" type=\\"application/json\\">{\\"abortReason\\":\\"!undefined\\",\\"_urlRewrite\\":null,\\"_urlRedirect\\":\\"!undefined\\",\\"abortStatusCode\\":\\"!undefined\\",\\"_abortCall\\":\\"!undefined\\",\\"_abortCaller\\":\\"!undefined\\",\\"_pageContextInitHasClientData\\":\\"!undefined\\",\\"_pageId\\":\\"/pages/index\\"}</script>"`;
